go/Dockerfile

```diff
@@ -1,8 +1,26 @@
-FROM alpine
+FROM golang:latest
 
-RUN addgroup -S mercari && adduser -S trainee -G mercari
-# RUN chown -R trainee:mercari /path/to/db
+# Set the working directory inside the container
+WORKDIR /go/app
 
-USER trainee
```

Comment:
If there is no reason, could you preserve using USER trainee?

see: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user